storybook/src/data/documents.data.js

```diff
@@ -0,0 +1,66 @@
+import uuid from 'uuid/v4';
+import image from './nuxeo-logo.jpg';
+
+export default class Document {
+  // class methods
+  constructor(type = '', properties = {}, facets = [], permissions = [], isTrashed = false, isFavorite = false) {
```

Comment:
Not a big fan of having hardcoded system properties as params here, since we can have more properties other than `isTrashed` and `isFavorite`. Perhaps we should have another for system properties?
```suggestion
  constructor(type = '', properties = {}, facets = [], permissions = [], systemProperties = {isTrashed: false, isFavorite: false}) {
```

Reply:
We should also add an object for contextParameters.